resources/user-agents/browsers/palemoon/palemoon-2-x.json

```diff
@@ -1,18 +1,18 @@
 {
-  "division": "Palemoon #MAJORVER#.#MINORVER#",
+  "division": "PaleMoon #MAJORVER#.#MINORVER#",
```

Comment:
It was confusing that the comment used a lower case "m" and the Browser used an uppercase "M" here, so I changed the comment and division headers to match the "Browser" property.